playwright/test/search-images.test.ts

```diff
@@ -28,41 +25,22 @@ test('(1) | Search by term, filter by colour, check results, view image details,
   await newSearch(context, page, 'images');
   await searchQuerySubmitAndWait('art of science', page);
 
-  if (isMobile(page)) {
-    await page.locator('button[aria-controls="mobile-filters-modal"]').click();
-    await expect(await page.locator('#mobile-filters-modal')).toBeVisible();
-    await page.getByTestId('swatch-red').click();
-    await page.getByRole('button', { name: 'Close modal window' }).click();
-  } else {
-    await page.locator('button[aria-controls="images.color"]').click();
-    await page.getByRole('button', { name: 'Red', exact: true }).click();
-    await page.locator('button[aria-controls="images.color"]').click();
-  }
-  await expect(
-    await page.getByTestId('image-search-results-container')
-  ).toBeVisible();
+  await selectAndWaitForColourFilter(page);
   await expect(
-    await page
-      .locator('[data-testid="image-search-results-container"] li')
-      .first()
+    page.getByTestId('image-search-results-container')
   ).toBeVisible();
   await clickActionClickSearchResultItem(1, page);
```

Comment:
Could we move this to the new helper file ([similarly to work's](https://github.com/wellcomecollection/wellcomecollection.org/blob/3caae3dea87ad93bf1bbea1cf3ec90de9240b490/playwright/test/helpers/search.ts#L67), maybe?) and get rid of it?

Reply:
I'll leave `.getByTestId('image-search-results-container')` hard coded (rather than pass in a testid) because the helper is specifically for `ClickSearchResultItem`

Reply:
specific to images results yes!

Reply:
Just saw the edited message after looking again and becoming confused myself 😓 I think I was looking at old code.

[It was added to the helper file](https://github.com/wellcomecollection/wellcomecollection.org/pull/10537/files#diff-e631726721f43d1e76bc3f8d2c3706f13a30d0da977aa00b93f6da0b0e9fe4e9) wasn't it but maybe I should've said nothing as I do think @georgiaewhitney is looking at making it better. Sorry!

Reply:
I added it anyway after updating the message, that was confusing sorry.
I moved it and imported it to use in the (1) test case, and I also left it in the test file because I didn't want to touch (2) and (3)

Reply:
So we just need to make sure @georgiaewhitney is aware and can remove it/modify it accordingly!